sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -228,6 +228,7 @@ def _management_request(self, mgmt_msg, op_type):
                     self._address.hostname, mgmt_auth
                 )  # pylint:disable=assignment-from-none
                 mgmt_client.open(connection=conn)
+                mgmt_msg.application_properties["security_token"] = mgmt_auth.get_token().token
```

Comment:
Doesn't this retrieve a different token to that which was used to authenticate the connection? Or does that not matter?

Reply:
Yes a different token is generated from the same credential. It's working though.
This is why I'm thinking of updating uamqp because uamqp may be able to reuse the token from the connection?

Reply:
@annatisch changed to use mgmt_auth.token directly so no new token is generated. It also works. Thank @yunhaoling for the suggestion.

Reply:
@YijunXieMS glad it helps, just don't forget to manually test this against the azure-identity 😃.
Or we could do one step further, we could add a few lines of code in the test_auth/test_auth_async to test?

Reply:
EventProcessor calls get_partition_ids( ) when it starts up. Every receive live test pass implies this works.